integrations/docker/images/chip-build-android/Dockerfile

```diff
@@ -1,11 +1,16 @@
 ARG VERSION=latest
 FROM connectedhomeip/chip-build:${VERSION}
 
-# JDK 11 access
+# JDK 11 & 8 access
+# JDK 8 is required because Android `sdkmanager`
+# doesn't work with JDK 11.
 RUN set -x \
     && apt-get update \
     && DEBIAN_FRONTEND=noninteractive apt-get install -fy \
     openjdk-11-jdk \
+    openjdk-8-jdk \
+    rsync \
+    swig \
```

Comment:
There are required when building the Android App the following the README doc, Please also see the document-update PR https://github.com/project-chip/connectedhomeip/pull/3334.